values-example.yaml

```diff
@@ -89,3 +89,103 @@ velero:
 
 openfaas:
   enabled: false
+
+# ClickHouse Vector Sink
+
+vector:
+  enabled: false
+  customConfig:
+    transforms:
+      # Events should be separated per namespace, and a different sink should be
+      # implemented for every namespace with Aspects
+      logs_openedx_demo:
+        type: filter
+        inputs:
+        - kubernetes_tutor_logs
+        condition: '.kubernetes.pod_namespace == "openedx_demo"' # Mkae sure to update the namespace
+
+      xapi_openedx_demo:
+        type: remap
+        inputs:
+        - logs_openedx_demo
+        drop_on_error: true
+        drop_on_abort: true
+        source: |-
+          parsed, err_regex = parse_regex(.message, r'^.* \[xapi_tracking\] [^{}]*
+          (?P<tracking_message>\{.*\})$')
+          if err_regex != null {
+            abort
+          }
+          message, err = strip_whitespace(parsed.tracking_message)
+          parsed_json, err_json = parse_json(parsed.tracking_message)
+          if err_json != null {
+            log("Unable to parse JSON from xapi tracking log message: " + err_json, level: "error")
+            abort
+          }
+          time, err_timestamp = parse_timestamp(parsed_json.timestamp, "%+")
+          if err_timestamp != null {
+            log("Unable to parse timestamp from tracking log 'time' field: " + err_timestamp, level: "warn")
+            time, err_timestamp = parse_timestamp(parsed_json.timestamp, "%+")
+            if err_timestamp != null {
+              log("Unable to parse timestamp from tracking log 'timestamp' field: " + err_timestamp, level: "error")
+              abort
+            }
+          }
+          event_id = parsed_json.id
+          . = {"event_id": event_id, "emission_time": format_timestamp!(time,
+          format: "%+"), "event": encode_json(parsed_json)}
+
+    sinks:
+      # Example ClickHouse Sink
+      clickhouse_openedx_demo:
+        type: clickhouse
+        auth:
+          strategy: basic
+          user: '{{ ASPECTS_CLICKHOUSE_VECTOR_USER }}'
+          password: '{{ ASPECTS_CLICKHOUSE_VECTOR_PASSWORD }}'
+        encoding:
```

Comment:
This are the tutor jinja variables right? I think in the context of a values file it might be a bit confusing considering the helm templates also use the brace syntax. Maybe you can use some example values and include in the comment each variable, like:

```
# Example ClickHouse Sink
# ASPECTS_CLICKHOUSE_VECTOR_USER: ch_vector_user
# ASPECTS_CLICKHOUSE_VECTOR_PASSWORD: hunter2
```